components/web/apache2-modules/mod_perl/apache-perl.p5m

```diff
@@ -289,95 +289,95 @@ file path=usr/apache2/2.4/lib/perl/auto/mod_perl2/.packlist
-file path=usr/apache2/2.4/man/man3/Apache2::Reload.3
-file path=usr/apache2/2.4/man/man3/Apache2::RequestIO.3
-file path=usr/apache2/2.4/man/man3/Apache2::RequestRec.3
-file path=usr/apache2/2.4/man/man3/Apache2::RequestUtil.3
-file path=usr/apache2/2.4/man/man3/Apache2::Resource.3
-file path=usr/apache2/2.4/man/man3/Apache2::Response.3
-file path=usr/apache2/2.4/man/man3/Apache2::ServerRec.3
-file path=usr/apache2/2.4/man/man3/Apache2::ServerUtil.3
-file path=usr/apache2/2.4/man/man3/Apache2::SizeLimit.3
-file path=usr/apache2/2.4/man/man3/Apache2::Status.3
-file path=usr/apache2/2.4/man/man3/Apache2::SubProcess.3
-file path=usr/apache2/2.4/man/man3/Apache2::SubRequest.3
-file path=usr/apache2/2.4/man/man3/Apache2::URI.3
-file path=usr/apache2/2.4/man/man3/Apache2::Util.3
-file path=usr/apache2/2.4/man/man3/Apache2::compat.3
-file path=usr/apache2/2.4/man/man3/Apache2::porting.3
-file path=usr/apache2/2.4/man/man3/Apache::Reload.3
-file path=usr/apache2/2.4/man/man3/Apache::SizeLimit.3
-file path=usr/apache2/2.4/man/man3/Apache::SizeLimit::Core.3
-file path=usr/apache2/2.4/man/man3/Apache::Test.3
-file path=usr/apache2/2.4/man/man3/Apache::TestConfig.3
-file path=usr/apache2/2.4/man/man3/Apache::TestHandler.3
-file path=usr/apache2/2.4/man/man3/Apache::TestMB.3
-file path=usr/apache2/2.4/man/man3/Apache::TestMM.3
-file path=usr/apache2/2.4/man/man3/Apache::TestReport.3
-file path=usr/apache2/2.4/man/man3/Apache::TestRequest.3
-file path=usr/apache2/2.4/man/man3/Apache::TestRun.3
-file path=usr/apache2/2.4/man/man3/Apache::TestRunPHP.3
-file path=usr/apache2/2.4/man/man3/Apache::TestRunPerl.3
-file path=usr/apache2/2.4/man/man3/Apache::TestServer.3
-file path=usr/apache2/2.4/man/man3/Apache::TestSmoke.3
-file path=usr/apache2/2.4/man/man3/Apache::TestTrace.3
-file path=usr/apache2/2.4/man/man3/Apache::TestUtil.3
-file path=usr/apache2/2.4/man/man3/Bundle::Apache2.3
-file path=usr/apache2/2.4/man/man3/Bundle::ApacheTest.3
-file path=usr/apache2/2.4/man/man3/ModPerl::BuildMM.3
-file path=usr/apache2/2.4/man/man3/ModPerl::CScan.3
-file path=usr/apache2/2.4/man/man3/ModPerl::Code.3
-file path=usr/apache2/2.4/man/man3/ModPerl::Config.3
-file path=usr/apache2/2.4/man/man3/ModPerl::Const.3
-file path=usr/apache2/2.4/man/man3/ModPerl::Global.3
-file path=usr/apache2/2.4/man/man3/ModPerl::MM.3
-file path=usr/apache2/2.4/man/man3/ModPerl::MethodLookup.3
-file path=usr/apache2/2.4/man/man3/ModPerl::PerlRun.3
-file path=usr/apache2/2.4/man/man3/ModPerl::PerlRunPrefork.3
-file path=usr/apache2/2.4/man/man3/ModPerl::Registry.3
-file path=usr/apache2/2.4/man/man3/ModPerl::RegistryBB.3
-file path=usr/apache2/2.4/man/man3/ModPerl::RegistryCooker.3
-file path=usr/apache2/2.4/man/man3/ModPerl::RegistryLoader.3
-file path=usr/apache2/2.4/man/man3/ModPerl::RegistryPrefork.3
-file path=usr/apache2/2.4/man/man3/ModPerl::Util.3
-file path=usr/apache2/2.4/man/man3/mod_perl2.3
+file path=usr/apache2/2.4/man/man3/APR.3perl
```

Comment:
... also these.  I believe they should go to `/usr/perl5/5.36/man/man3perl/` with (probably mediated) symlinks from `/usr/share/man/man3perl/`.

Reply:
Again ist is something belonging to Apache - not perl. And it would be a complete change to what is was before.